presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnectionConfig.java

```diff
@@ -39,26 +49,33 @@
 
     private final HostAndPort address;
     private final URI uri;
+    private final Properties connectionProperties;
 
     private String catalog;
     private String schema;
 
     private final boolean useSecureConnection;
 
-    public PrestoDriverUri(String url)
+    private static final Map<ConnectionProperty, BiConsumer<HttpClientConfig, String>> CLIENT_SETTERS = ImmutableMap.of(
```

Comment:
btw, what is the reason to have `BiConsumer` that holds the method references? Why can't we just explicitly call this methods of `HttpClientConf` in HttpClient factory method somewhere based on the `HttpClientProperties` POJO?